server/datastore/mysql/software.go

```diff
@@ -2614,6 +2640,12 @@ INNER JOIN software_cve scve ON scve.software_id = s.id
 			for _, path := range installedPaths {
 				ver := bySoftwareID[path.SoftwareID]
 				ver.InstalledPaths = append(ver.InstalledPaths, path.InstalledPath)
+				if ver.Source == "apps" {
```

Comment:
"apps" covers homebrew-installed etc.?

Reply:
Does not. Only stuff returned by the `apps` osquery table.

Reply:
Ah cool they do show up! So, we are good here :)

Reply:
That said, all my `.app`s under homebrew are not signed... (empty `team_identifier`)